components/Link.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import NextLink from 'next/link'
+
+export default function Link({ url, children }) {
+    return (
+        <NextLink
+            href={url}
+            target="_blank"
```

Comment:
I don't think every link should open on a new page

Reply:
agreed, i can add a prop to allow for customization of the target